internal/helpers/helpers.go

```diff
@@ -0,0 +1,11 @@
+package helpers
+
+// removes an argument and its value from the commandArgs slice
+func RemoveArg(args []string, arg string) []string {
+	for i, v := range args {
+		if v == arg {
+			return append(args[:i], args[i+2:]...)
+		}
+	}
+	return args
+}
```

Comment:
```suggestion
// RemoveStringFromSlice removes the provided element if found
func RemoveStringFromSlice(args []string, arg string) []string {
	return slices.DeleteFunc(args, func(str string) {
	    return str == arg
	})
}
```
Let's make this a bit more generic in name, the args use case is a bit specific for this to go into a helpers package. I also don't know how I feel about a generic helpers/utils package yet. Maybe make our own `slices` package with has a `DeleteString` function in it. Maybe even in this case since it is only used in 2 places, just call the stdlib slices package directly instead of a new helper/package.

Reply:
Hmm, unless you are able to specify the value as well as the key. You could also use `slices.Delete` and delete +2 like you were previous